public/js/core.js

```diff
@@ -37,12 +37,14 @@ app.controller('mainController', function ($scope, $state, Todo) {
 
     $scope.createTodo = function () {
         $scope.loading = true;
-
-        if ($scope.newTodo.text !== '') {
+        if ($scope.newTodo.text !== "") {
             Todo.create($scope.newTodo)
                 .success(function (data) {
-                    $scope.newTodo = { text: '' };
-                    $scope.todos = data;
+                    $scope.newTodo = { text: "" };
+
+                    var todos = $scope.todos;
```

Comment:
This is redundant. You can use it for simplify and minize code line width if you have many usages. But here only one usage I see